server.js

```diff
@@ -1,20 +1,125 @@
 const express = require("express");
 const cors = require("cors");
+const uuid = require("uuid");
+const moment = require("moment");
+const validator = require("email-validator");
 
 const app = express();
 
 app.use(express.json());
+app.use(express.urlencoded({ extended: true}));
 app.use(cors());
 
 //Use this array as your (in-memory) data store.
 const bookings = require("./bookings.json");
 
+
+function isInvalidId(id, index, response) {
+  if (index < 0) {
+    response.status(400).send("No booking with Id: " + id + " is found");
+  }
+}
+
+
 app.get("/", function (request, response) {
   response.send("Hotel booking server.  Ask for /bookings, etc.");
 });
 
 // TODO add your routes and helper functions here
 
+// GET All Bookings
+
+app.get("/bookings", function(req, res) {
+  if(bookings.length <= 0) {
+    res.status(500).send("No Available Data");
```

Comment:
We only send 500 Internal Server error status when there is something unexpected happening on the server. Having no bookings is not unexpected! You could just send back an empty array.

Reply:
Okay. Thank you so much!